examples/incrementImage/increment.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+class Increment extends React.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  /* iterate over canvas and set each pixel as random */
+  generatePixels(canvas, ctx, canvasData, incr) {
```

Comment:
That was just me thinking of ideas for increment, didn't realize I pushed it. The whole directory can be deleted